Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Build Cruise Control in its own stage
+FROM amazoncorretto:21-alpine-jdk AS build
+WORKDIR /workspace
+COPY . .
+RUN ./gradlew clean jar copyDependantLibs --warning-mode all
+
+# Fetch the jars, configs, and the startup script and run Cruise Control
+FROM amazoncorretto:21-alpine AS runtime
+RUN apk add --no-cache bash
+WORKDIR /cc
+COPY --from=build /workspace/cruise-control/build/ /cc/cruise-control/build/
+COPY --from=build /workspace/config/ /cc/config/
+COPY --from=build /workspace/observability/ /cc/observability/
```

Comment:
Important: This particular line depends on https://github.com/linkedin/cruise-control/pull/2351 to be merged first. 

We should remove it in case this PR is approved first; and then add it back afterwards when #2351 is merged.